Dockerfile.server

```diff
@@ -0,0 +1,13 @@
+FROM rust:latest as builder
+WORKDIR /usr/app/src
+COPY ./ /usr/app/src
+RUN apt-get update && apt-get -y install clang && \
+    rm -rf /var/lib/apt/lists/* && \
+    rustup component add rustfmt && \
+    cargo build --lib --bin indradb-server --release
+
+FROM debian:stretch-slim
+WORKDIR /root
+COPY --from=builder /usr/app/src/target/release/indradb-server .
+EXPOSE 27615
+ENTRYPOINT ["./indradb-server"]
```

Comment:
I _think_ (but not sure) you could combine these Dockerfiles into one, and create multiple images; e.g. [see here](https://adilsoncarvalho.com/creating-multiple-images-from-a-single-dockerfile-3f69254b6137)